packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -199,64 +176,54 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
           formSection({
             label: t('account.about_you'),
             editButton: t('account.edit_information'),
-            onSubmit: (values) => {
-              const consents = formatConsentsFromValues(publisherConsents, { ...values.metadata, ...values.consentsValues });
-
-              return accountController.updateUser({
-                firstName: values.firstName || '',
-                lastName: values.lastName || '',
-                metadata: {
-                  ...values.metadata,
-                  ...formatConsentsToRegisterFields(consents),
-                  consents: JSON.stringify(consents),
-                },
-              });
+            onSubmit: async (values) => {
+              if (registrationFields) {
+                await accountController.updateRegistrationFieldsValues(registrationFields.fields, values.registrationFieldsValues);
+                return;
+              }
+              throw new Error('Registration fields are not loaded');
```

Comment:
Should be aligned with my updates to useForm, to use FormValidationError, but you've mentioned that already.

Reply:
Yes, correct. We need these branches aligned asap 😄